src/helpers/handleError.ts

```diff
@@ -0,0 +1,17 @@
+import Express from 'express'
+
+interface HttpError {
```

Comment:
lets make a separate folder for the interfaces and contracts. Because we will be using these quite often.

I prefer having a folder `contracts` where we can define all the types  (data exchanges from FE, database and generic types)

Reply:
I'd prefer to have the related interfaces beside their implementers as much as we can, and maybe for the more general interfaces or types, we can have a separate folder for them. 

That's said because, we will have a lot of interfaces, it would be too hard to locate a specific one if we put all of them in a single folder, what's your opinion?

Reply:
For the errors and function params, yes we can keep in the same file. but something like suppose `IconsModel` which is constant through the app, I would have it like `import {IconsModel} from '@/Contracts/Database`, and similarly for rest of the common interfaces.